backstopConfig.js

```diff
@@ -37,6 +37,7 @@ const config = {
       ...basic,
       label: 'Header tag',
       selectors: ['header'],
+      misMatchThreshold: 5
```

Comment:
The `misMatchThreshold` is set to `5`, which might be too high depending on the project's requirements for visual testing. Consider adjusting this value to a more appropriate threshold if necessary.